src/main/java/uk/gov/justice/probation/courtcaseservice/service/ImmutableCourtCaseService.java

```diff
@@ -40,7 +41,22 @@ public CourtCaseEntity createCase(String courtCode, String caseNo, CourtCaseEnti
                         },
                         () -> trackCreateEvents(updatedCase));
 
-        return courtCaseRepository.save(updatedCase);
+        return Mono.just(updatedCase)
+            .map(courtCaseRepository::save)
+            .doAfterTerminate(() -> updateStatusForCrn(updatedCase.getCrn(), updatedCase.getProbationStatus(), updatedCase.getCaseNo()));
+    }
+
+    void updateStatusForCrn(String crn, String probationStatus, String caseNo) {
+        final var courtCases = courtCaseRepository.findOtherCurrentCasesByCrn(crn, caseNo)
+            .stream()
+            .filter(courtCaseEntity -> !courtCaseEntity.getProbationStatus().equalsIgnoreCase(probationStatus))
```

Comment:
Referring back to my earlier comment about the 'active' case being updated - this will make sure it isn't updated twice so happy for that to just be a name change.

Reply:
☝️ Ignore this - previous comment wasn't valid